pages/index.js

```diff
@@ -14,6 +14,40 @@ import includes from 'lodash/includes';
 import Link from 'next/link';
 
 export default class extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      // default to changelog if no forumlink is found
+      releaseNote: {
+        link: '/changelog'
+      }
+    }
+  }
+
+  componentDidMount() {
+    fetch(`https://forums.resin.io/c/etcher.json`)
+    .then(res => res.json())
+    .then(topics => {
+      const JVIOTTI_USERID = 4;
+      return topics.topic_list.topics.find(topic => {
+        // check if jviotti made the post
+        // and it contains latest version
+        return (
+          topic.title.includes(`Etcher ${locals.version} release`) &&
+          topic.posters[0].user_id == JVIOTTI_USERID
+        )
+      })
+    })
+    .then(releaseNote => {
+      if (releaseNote) {
+        releaseNote.link = `https://forums.resin.io/t/${releaseNote.slug}`;
```

Comment:
Is it worth making a constant for the `https://forums.resin.io` part of the two URLs used here? \*shrug\*